lms/static/sass/course/instructor/_instructor_2.scss

```diff
@@ -1541,6 +1541,7 @@
       background: white;
       padding: 5px;
       box-shadow: 0 3px 6px rgba(0, 0, 0, 0.16), 0 3px 6px rgba(0, 0, 0, 0.23);
+      z-index: 2;
```

Comment:
Why z-index 2? Is it relevant to other z-index?

Reply:
By default it has the `z-index: auto` property and when we set `z-index:1` it faded out something like this.  
<img width="705" alt="screen shot 2018-08-15 at 5 16 42 pm" src="https://user-images.githubusercontent.com/7627421/44147793-0e18dda8-a0af-11e8-82b6-2a74c64897d9.png">

On this index, it working fine.